MANIFEST.in

```diff
@@ -11,17 +11,7 @@
 #
 #  check-manifest -u
 
-include .editorconfig
```

Comment:
Basically MANIFEST.in specifies what else goes into a source distribution. I don't know yet if including tests makes a lot of sense, but I am leaving it as is since it seems prevalent in the python world. (removals are for file types not in this project).

Reply:
Some more info for the curious: https://packaging.python.org/en/latest/guides/using-manifest-in/

Reply:
Really good to point out. I've added a reference to it in the comments.